### PR TITLE
Implement Issue #1: Allow transport rides to remain free

### PR DESCRIPTION
**Pull request by Sadret**
Changes:
- added option to ignore free rides
- added license hint
- added co-author
- changed TS reference path to relative path
- updated Readme

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -9,12 +9,15 @@ const main = function (): void {
...
 };
 
 registerPlugin({
     name: 'Ride Price Manager',
-    version: '1.0',
-    authors: ['MarkG'],
+    version: '1.1',
+    authors: ['MarkG', 'Sadret'],
     type: 'remote',
+    licence: 'MIT',
```
Comment (mgovea) [inline src/index.ts:21]: sick, thanks

**Comment by mgovea** [comment]
Eh, I'll do my nits later.

**Comment by Sadret** [comment]
Sorry that I didn't reply, I wasn't notified by your comments.
Thanks for fixing the remaining issues and merging my proposal.

**Comment by mgovea** [comment]
Yeah np. Thanks for making a PR using such a simple and elegant solution! :)

**Pull request merged**